### PR TITLE
Fixed typos

### LINKED ISSUE #2561: a animal -> an animal
https://github.com/exercism/java/blob/8b8b73da1cd5c2a3e3de2cbe44d2e8de8e7ab279/exercises/concept/wizards-and-warriors/.docs/introduction.md?plain=1#L16

### PR DESCRIPTION
**Pull request by smicaliz**
Closes #2561 

I checked and fixed some other typos too.

### CODE REVIEW

File: exercises/concept/wizards-and-warriors/.docs/introduction.md
Comment (sanderploegsma) [inline exercises/concept/wizards-and-warriors/.docs/introduction.md:1]: This file is actually generated by the `introduction.md.tpl` template file, so you should make these changes in the source document instead and then regenerate this file.

The source file is `concepts/inheritance/introduction.md`, and you can use the configlet to regenerate the file:

```sh
bin/configlet generate
```

**Review by sanderploegsma — CHANGES_REQUESTED** [review]
One thing to take care of, the rest looks good! 👍

File: concepts/inheritance/introduction.md [hunk 1]
```diff
@@ -36,7 +37,7 @@ Now whenever we do,
 
 ```java
 Animal animal = new Lion(); //creating instance of Animal, of type Lion
-animal.bark();
+        animal.bark();
```
Comment (sanderploegsma) [inline concepts/inheritance/introduction.md:40]: What is the reason for adding this indentation? I don't think it makes the snippet more readable. Could you revert it please?

**Review by sanderploegsma — CHANGES_REQUESTED** [review]

**Comment by smicaliz** [comment]
Actually I did not any of that 2 modifications. I think there was something auto or I didn't pay enough attention. I should be more attentive. It's fixed. Thanks the review!

**Review by sanderploegsma — APPROVED** [review]
LGTM 👍 Thanks for the fix!

**Pull request merged**